common/network-shuffle/src/main/java/org/apache/spark/network/shuffle/OneForOneBlockFetcher.java

```diff
@@ -100,7 +114,14 @@ public void onSuccess(ByteBuffer response) {
           // Immediately request all chunks -- we expect that the total size of the request is
           // reasonable due to higher level chunking in [[ShuffleBlockFetcherIterator]].
           for (int i = 0; i < streamHandle.numChunks; i++) {
-            client.fetchChunk(streamHandle.streamId, i, chunkCallback);
+            if (fetchToDisk) {
+              final File targetFile = new File(".",
```

Comment:
Yes, I wanted to use `DiskBlockManager.getFile`, but I found it's hard to import `DiskBlockManager` from `OneForOneBlockFetcher`.